It seems that `CI Build / Backport packages` is still failing with this error:
```
ERROR: There were some import errors

----------------------------------------
Traceback (most recent call last):
  File "/import_all_provider_classes.py", line 61, in import_all_provider_classes
    _module = importlib.import_module(module_name)
  File "/usr/local/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/local/lib/python3.6/site-packages/airflow/providers/amazon/aws/hooks/ses.py", line 24, in <module>
    from airflow.utils.email import build_mime_message
ImportError: cannot import name 'build_mime_message'

----------------------------------------
```

I'm not sure what I'm missing here 🤔

--- a/backport_packages/refactor_backport_packages.py
+++ b/backport_packages/refactor_backport_packages.py
@@ -430,6 +430,17 @@ class RefactorBackportPackages:
             rename("airflow.providers.amazon.common.utils.typing_compat")
         )
 
+        copyfile(
+            os.path.join(get_source_airflow_folder(), "airflow", "utils", "email.py"),
+            os.path.join(get_target_providers_package_folder("amazon"), "common", "utils", "email.py")
+        )
+        (
+            self.qry.
+            select_module("airflow.utils.email").
+            filter(callback=amazon_package_filter).
+            rename("airflow.providers.amazon.common.email")
+        )
+
     def refactor_google_package(self):
         r"""
         Fixes to "google" providers package.

--- a/backport_packages/refactor_backport_packages.py
+++ b/backport_packages/refactor_backport_packages.py
@@ -438,7 +438,7 @@ class RefactorBackportPackages:
             self.qry.
             select_module("airflow.utils.email").
             filter(callback=amazon_package_filter).
-            rename("airflow.providers.amazon.common.email")
+            rename("airflow.providers.amazon.common.utils.email")
         )
 
     def refactor_google_package(self):

